trino-connector/src/test/java/com/datastrato/gravitino/trino/connector/TestGravitinoConnector.java

```diff
@@ -214,7 +214,7 @@ public void testCreateCatalog() throws Exception {
     assertThat(computeActual("show catalogs").getOnlyColumnAsSet()).contains("memory");
 
     // testing the gravitino connector framework works.
-    assertThat(computeActual("select * from system.jdbc.tables"));
+    assertThat(computeActual("select * from system.jdbc.tables").getRowCount()).isEqualTo(60);
```

Comment:
This may be a bit fragile, is the answer always going to be 60? Perhaps just testing that it returns more than one row is good enough?

Reply:
Will change accordingly